frontend/src/components/facilities/FacilityLocationAttributes.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <v-card variant="outlined" class="pa-4">
+    <AppLabel>Select all that apply for your facility:</AppLabel>
+    <div class="mt-4">
+      <div>Is your facility located on K-12 school grounds or affiliated with a Board of Education?</div>
+      <v-radio-group id="k12-school-grounds" v-model="model.k12SchoolGrounds" :rules="rules.required" :hide-details="readonly" :disabled="readonly" inline color="primary">
+        <v-radio label="Yes" :value="1" />
+        <v-radio label="No" :value="0" />
+      </v-radio-group>
+    </div>
+    <div>
+      <div>Is your facility located in a municipal community center?</div>
+      <v-radio-group id="municipal-community" v-model="model.municipalCommunity" :rules="rules.required" :hide-details="readonly" :disabled="readonly" inline color="primary">
+        <v-radio label="Yes" :value="1" />
+        <v-radio label="No" :value="0" />
+      </v-radio-group>
+    </div>
+    <div>
+      <div>Is your facility located on Reserve?</div>
+      <v-radio-group id="reserve" v-model="model.reserve" :rules="rules.required" :hide-details="readonly" :disabled="readonly" inline color="primary">
+        <v-radio label="Yes" :value="1" />
+        <v-radio label="No" :value="0" />
+      </v-radio-group>
+    </div>
+    <div>
+      <div>
+        Is your facility designated as a
+        <a href="https://www2.gov.bc.ca/gov/content/family-social-supports/caring-for-young-children/child-care-funding/young-parent-program" target="_blank">Young Parent Program</a>
```

Comment:
As mentioned let's make sure we get confirmation on this link. If there's some uncertainty we could move it to a environment var but that's probably overkill for a static link.

Reply:
it's updated